include/ck/tensor_operation/gpu/grid/gridwise_put_element_1d.hpp

```diff
@@ -0,0 +1,156 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2018-2022, Advanced Micro Devices, Inc. All rights reserved.
+
+#pragma once
+
+#include "ck/tensor_description/cluster_descriptor.hpp"
```

Comment:
Remove this including since it is not needed

Reply:
4f1dbdf5a3bb0abd0f34dd284631ced9b2919226